app/src/main/java/com/cornellappdev/uplift/networking/UpliftApiRepository.kt

```diff
@@ -77,6 +78,7 @@ object UpliftApiRepository {
                     }
                 }
                 .catch {
+                    Log.e("query", it.stackTraceToString())
```

Comment:
I'll keep the error logs since I keep needing to go back and add them for like every time I face a problem, so I'm sure we'll need em again in the future lol. I'll delete the purely debugging ones, though.